SIGSEGV on endspoints list

hi! 

I faced with problem trying to list endpoints from my swagger-file. 

```shell
$ ./swama endpoints list -f ~/Downloads/swagger\ \(2\).json
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x7a66bf]

goroutine 1 [running]:
github.com/idsulik/swama/internal/swagger.(*endpoints).ListEndpoints(0xc00027bc48, {0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, {0x8c95fa, 0x3})
	/home/bagger/src/swama/internal/swagger/endpoints.go:71 +0xc3f
github.com/idsulik/swama/cmd/endpoints.newListCommand.func1(0xc00018e400?, {0x8c9862?, 0x4?, 0x8c9866?})
	/home/bagger/src/swama/cmd/endpoints/list.go:36 +0xac
github.com/spf13/cobra.(*Command).execute(0xc000198c08, {0xc00021c140, 0x2, 0x2})
	/home/bagger/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:985 +0xaaa
github.com/spf13/cobra.(*Command).ExecuteC(0xc57840)
	/home/bagger/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/bagger/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/home/bagger/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1034
github.com/idsulik/swama/cmd.Execute(...)
	/home/bagger/src/swama/cmd/root.go:27
main.main()
	/home/bagger/src/swama/main.go:24 +0x11c
```

I tried different openapi files but it seems none of them works - the error always the same. 
I'm attaching example swagger file. [swagger (2).json](https://github.com/user-attachments/files/17019383/swagger.2.json)

My swama installation was compiled from source codes.  

Could you tell please what is wrong with my installation?

--- a/internal/swagger/endpoints.go
+++ b/internal/swagger/endpoints.go
@@ -67,9 +67,13 @@ func (e *endpoints) ListEndpoints(method, endpoint, tag, group string) error {
 			if group != "" {
 				keys := make([]string, 0)
 				if group == GroupByTag {
-					for _, tag := range operation.Tags {
-						description := e.doc.Tags.Get(tag).Description
-						keys = append(keys, fmt.Sprintf("%s (%s)", tag, description))
+					for _, tagName := range operation.Tags {
+						tag := e.doc.Tags.Get(tagName)
+						key := tagName
+						if tag != nil {
+							key += fmt.Sprintf(" (%s)", tag.Description)
+						}
+						keys = append(keys, key)
 					}
 				} else if group == GroupByMethod {
 					keys = append(keys, m)

